styles/colors-silicon-night.less

```diff
@@ -50,7 +51,8 @@
 // Checkbox
 // ---------------------------------------
 @agate-checkbox-text-color: @agate-white;
-@agate-checkbox-bg-color: @agate-accent;
+@agate-checkbox-border-color: rgba(255, 255, 255, 0.5);
```

Comment:
`fade` function here too: `fade(@agate-foreground, 50%)` (will work correctly after changing the definition above). In fact, you could set this `fade(@agate-foreground, 50%)` rule in the day file (replacing the checkbox border color line) and omit it from here, since the relationship is the same in night mode. Neato.

Now that I look more deeply, updating the @agate-foreground rule mentioned above would remove the need to set all of these checkbox values, since they can just use the conceptual color values.